storage/Devices/BlkDevice.cc

```diff
@@ -48,7 +48,10 @@ namespace storage
 		return blk_device;
 	}
 
-	throw DeviceNotFound(sformat("device not found, name:%s", name.c_str()));
+	ST_THROW(DeviceNotFound(sformat("device not found, name:%s", name.c_str())));
```

Comment:
OK, I'm used to that style from the old libstorage. There "key1:value1 key2:value2" is used. I'm fine with using "key1: value1, key2: value2" in the future.